src/js/components/FormField/__tests__/FormField-test.tsx

```diff
@@ -158,6 +160,39 @@ describe('FormField', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test.only('does not apply focusIndicator when formField.focusIndicator is false', () => {
+    const { container } = render(
+      <Grommet
+        theme={{
+          formField: {
+            focusIndicator: false,
+          },
+        }}
+      >
+        <FormField htmlFor="checkbox-id">
+          <CheckBox label="CheckBox" />
+        </FormField>
+        <FormField htmlFor="radio-id">
+          <RadioButtonGroup
+            name="radio-group"
+            options={['Option 1', 'Option 2']}
+          />
+        </FormField>
+        <FormField htmlFor="range-id">
+          <RangeInput name="range" min={0} max={100} />
+        </FormField>
+      </Grommet>,
+    );
+
+    // Simulate focus
+    fireEvent.focus(screen.getByRole('checkbox'));
+    fireEvent.focus(screen.getByLabelText('Option 1'));
+    fireEvent.focus(screen.getByLabelText('Option 2'));
+    fireEvent.focus(screen.getByRole('slider'));
+
+    expect(container).toMatchSnapshot();
```

Comment:
Ill look back at the test see if I can check directly for border to be more accurate